ting_subsearch_secondary.module

```diff
@@ -119,25 +120,21 @@ function ting_subsearch_secondary_execute_secondary_search($keys, $results) {
     'profile' => variable_get('ting_subsearch_secondary_profile', ''),
   ]);
 
-  if ($secondary_result->numTotalObjects > variable_get('ting_subsearch_secondary_result_min', 100)) {
+  if ($secondary_result->numTotalObjects > variable_get('ting_subsearch_secondary_result_min', 100) && $secondary_result->numTotalObjects < variable_get('ting_subsearch_secondary_max', 100)) {
     $ratio = $secondary_result->numTotalObjects / $results->getNumTotalObjects();
+
     if ($ratio > variable_get('ting_subsearch_secondary_ps_factor', 2)) {
       $items = [];
       $number_of_records_to_show = variable_get('ting_subsearch_secondary_number_of_results', 4);
       foreach ($secondary_result->collections as &$collection) {
         $build = ting_collection_view($collection, 'minimal');
         $markup = drupal_render($build);
-        $return = l($markup, 'https://bibliotek.dk/work/' . $collection->ding_entity_id, [
```

Comment:
Where was this moved to @bozeak ?

Reply:
This was removed because it was not used. All magic happens in `$items[]`.